### DISCUSSION TITLE
Incorrect activity label in explore on mobile Android

### DISCUSSION THREAD

**Discussion by robertvanhoesel** [community/community#27811] [Mobile] [answered]
Someone starring a repository has the translation string / label "{user} posted a discussion" vs "{user} stared a repository"

**Comment by aconsuegra** [comment, marked as answer]
Hi @robertvanhoesel, thanks for reaching our and for the bug report! We have fixed the issue and it will be included in this week's update, which will start rolling out in Google Play tomorrow.

**Reply by Thaonhi1998** [reply]
https://github.com/Thaonhi1998/Thaonhi1998/new/main

**Comment by Thaonhi1998** [comment]
> Someone starring a repository has the translation string / label "{Thaonhi1998} posted a discussion" vs "{Thaonhi1998} stared a repository"
